README.md

```diff
@@ -329,17 +329,23 @@ cd build
 cmake -GNinja ../ -DSYCL_COMPILER=dpcpp
 ninja
 ```
+The target triplet can be set by adding `-DDPCPP_SYCL_TARGET=<triplet>`. If it
+is not set, the default values is `spir64_x86_64-unknown-unknown`, which targets
```

Comment:
I think we should change the default target to - spir64-unknown-unknown to be consistent with what DPCPP specifies: 
`spir64 - this is the default generic SPIR-V target;`

Reply:
I did it in the last commits.